packages/cli/src/bin.ts

```diff
@@ -27,7 +27,7 @@ import { DebugInfo } from './DebugInfo'
 import { Format } from './Format'
 import { Generate } from './Generate'
 import { Init } from './Init'
-import { PlatformCommand } from './platform/PlatformCommand'
+import { Platform } from './platform'
```

Comment:
I like the idea of namespacing all the commands instead of following the pattern `<NAMESPACE><COMMAND>`, for example `MigrateDev`. I would like to hear @Jolg42 opinion on this one as well.

Would `PlatformCommand` be a better name? That reads quite nicely and communicates the intention of wrapping all commnds for the platform. Examples:
 - PlatformCommand.Auth.Login
 - PlatformCommand.Project.Create

Reply:
I'm not sure I follow the pattern. `PlatformCommand` but not `AuthPlatform` etc. What makes those namespaces different?